support-frontend/assets/components/priceCards/priceCardsContainer.tsx

```diff
@@ -49,11 +52,12 @@ export function PriceCardsContainer({
 	);
 	const minAmount = useContributionsSelector(getMinimumContributionAmount());
 
-	const inThreeTierVariant = inThreeTierV2Variant(
+	const inThreeTierVariant = inThreeTierV3Variable(
 		useContributionsSelector((state) => state.common.abParticipations),
 	);
 	const tierBillingPeriod =
 		paymentFrequency === 'ANNUAL' ? 'annual' : 'monthly';
+	const tierCards = inThreeTierVariant ? tierCardsVariable : tierCardsFixed;
```

Comment:
`inThreeTierVariant` is going to be `true` if the user is in `variantFixed` OR `variantVariable` as
inThreeTierV3Variable returns `true` in both these scenarios - do we want `tierCards` to be assigned the value of`tierCardsVariable` even if a user is in `variantFixed`?